mlflow/models/utils.py

```diff
@@ -50,25 +51,33 @@ class _Example(object):
           encoded strings.
         - numpy types: Numpy types are converted to the corresponding python types or their closest
           equivalent.
+        - csc/csr matric: similar to 2 dims numpy array, csc/csr matric are converted to
+          corresponding python types or their closest equivalent.
     """
 
     def __init__(self, input_example: ModelInputExample):
         def _is_scalar(x):
             return np.isscalar(x) or x is None
 
         def _is_tensor(x):
-            return isinstance(x, np.ndarray) or (
-                isinstance(x, dict) and all([isinstance(ary, np.ndarray) for ary in x.values()])
+            return (
+                isinstance(x, np.ndarray)
+                or (
+                    isinstance(x, dict) and all([isinstance(ary, np.ndarray) for ary in x.values()])
+                )
+                or isinstance(x, (csr_matrix, csc_matrix))
```

Comment:
Can we merge this and `isinstance(x, np.ndarray)` into isinstance(x, (np.ndarray, csr_matrix, csc_matrix))?